SU-SWS/stanford_profile
# D8CORE-1659 Set image metatags to use an image style

# READY FOR REVIEW

# Summary
- Set image metatags to use an image style

# Need Review By (Date)
- 3/26

# Urgency
- low

# Steps to Test
1. checkout this branch
1. drush cim -y
1. view source on a page with a banner field
1. verify the metatag `og:image` uses an image style

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
